packages/core/redux/mainWallet/actions.js

```diff
@@ -467,7 +467,9 @@ export const createNewChildAddress = ({ blockchain, tokens, name, deriveNumber }
   wallets
     .items()
     .map((wallet) => {
-      if (wallet.owners().items().filter((owner) => owner.address() === account).length > 0 && wallet instanceof DerivedWalletModel) {
+      if (wallet.owners().items().filter((owner) => {
+        return owner.address() === account
+      }).length > 0 && wallet instanceof DerivedWalletModel) {
```

Comment:
Please use 'wallet instanceof DerivedWalletModel' condition in the beginnning. 
If wallet is not instance of DerivedWalletModel, then we should not go through its owners etc.